packages/front-end/components/Archetype/SimulateFeatureValues.tsx

```diff
@@ -0,0 +1,516 @@
+                            feature.id + selectedEnvironment
+                          );
+                          if (!res) {
+                            return <td>-</td>;
+                          }
+                          return (
+                            <td>
+                              <div>
+                                {ArchetypeValueDisplay({
+                                  result: res,
+                                  feature,
+                                })}
+                              </div>
+                            </td>
+                          ); // Replace this with what you want to render
+                        })()
+                      ) : (
+                        <>
+                          {environments.slice(0, maxEnvironments).map((en) => {
+                            const res = featureResultsMap.get(
+                              feature.id + en.id
+                            );
+                            if (!res) {
+                              return (
+                                <td key={"unknown-" + en.id + feature.id}>-</td>
+                              );
+                            }
+                            return (
+                              <td
+                                key={feature.id + en.id + "row"}
+                                className="position-relative  cursor-pointer"
+                              >
+                                <div>
+                                  {ArchetypeValueDisplay({
+                                    result: res,
+                                    feature,
+                                  })}
+                                </div>
+                              </td>
+                            );
+                          })}
+                        </>
+                      )}
+                    </tr>
+                  </Fragment>
+                );
+              })}
+              {!items.length && (
+                <tr>
+                  <td colSpan={numColumns}>No matching features</td>
+                </tr>
+              )}
+            </tbody>
+          </table>
+          {Math.ceil(items.length / NUM_PER_PAGE) > 1 && (
+            <Pagination
+              numItemsTotal={items.length}
+              currentPage={currentPage}
+              perPage={NUM_PER_PAGE}
+              onPageChange={(d) => {
+                setCurrentPage(d);
+              }}
+            />
+          )}
+        </div>
+      </div>
+      <div className="alert-info mt-5 p-3 cursor-pointer align-items-center">
+        <div
+          className="d-flex"
+          onClick={(e) => {
+            e.preventDefault();
+            setOpenWarning(!openWarning);
+          }}
+        >
+          <div className="p-2 pr-3">
+            <FiAlertTriangle />
+          </div>
+          <div>
+            These results use the JS SDK, which supports the V2 hashing
```

Comment:
Do we need this text? It's confusing to me.
- Why is this a warning? Do I need to change something?
- What are the older/unsupported SDKs? Am I using one?
- What experiment? These are features... (presumedly linked experiments, but its not obvious)

And I don't think we're actually answering the important question here:
- What hash version was used to evaluate these features? Did we force v1 or v2? Or did we use the per experiment setting?

Reply:
The hash can be set per experiment/rollout. We don't force it one way or the other, but it can be v2, and the SDK they use might not support it.